### DISCUSSION TITLE
It is possible to set as Section child a TextRun without using a Paragraph

### DISCUSSION THREAD

**Discussion by sentientmachin3** [dolanmiu/docx#1642] [General]
I don't think this is an issue, but I find it useful to highlight it in case someone else encounters this behaviour. **Accidentally** I passed as Section child a TextRun instead of a paragraph:

```
this.doc = new Document({
      sections: [
        {
          children: [new TextRun({ text: 'Hello world' })],
        },
      ],
    })
```
The compiler did not complain about it, so I tried it and the document was just a blank page. According to the documentation and the type definition we are supposed to pass `(Paragraph | Table | TableOfContents)[]` as `children`. 
Passing a `Paragraph` did solve the "issue".

Versions: TS 4.7.2, library version 7.4.1

I don't know why the compiler does not complain, any explanations?

**Comment by dolanmiu** [comment]
Not possible sorry, 

Every TextRun needs to be in a paragraph

Unfortunately, thats simply how Word works
